hashie_rails.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.files = Dir['{app,config,db,lib}/**/*', 'MIT-LICENSE', 'Rakefile', 'README.rdoc']
   s.test_files = Dir['test/**/*']
 
-  s.add_dependency 'rails', '~> 4.0'
+  s.add_dependency 'activesupport', '~> 4.2.0'
```

Comment:
I don't think you wanted to change the dependency to be a 4.2.0, because now anyone on Rails 4.1.x can't use this. No?